config/src/main/kotlin/org/springframework/security/config/annotation/web/AuthorizeRequestsDsl.kt

```diff
@@ -54,7 +54,7 @@ class AuthorizeRequestsDsl : AbstractRequestMatcherDsl() {
      * Adds a request authorization rule for an endpoint matching the provided
      * pattern.
      * If Spring MVC is on the classpath, it will use an MVC matcher.
-     * If Spring MVC is not an the classpath, it will use an ant matcher.
+     * If Spring MVC is not on the classpath, it will use an ant matcher.
```

Comment:
Thanks for the cleanup. Will you please move these cleanups to a separate commit so that they are easier to backport to earlier versions?